src/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -439,8 +444,7 @@ private enum SequenceType
             MultiSegment = 0x00,
             Array = 0x1,
             MemoryManager = 0x2,
-            String = 0x3,
-            Empty = 0x4
+            String = 0x3
```

Comment:
Seems to me that when _sequenceStart and _sequenceEnd is negative value 4 will still be possible. I do not know much about this code to tell if this is realistic combination though